### DISCUSSION TITLE
about import module

### DISCUSSION THREAD

**Discussion by timi5691** [vlang/v#21495] [General]
in A module, i import math module.
in Main module, i import A module.

How i can use math module's functions and types in main module without have to import it again?

**Comment by JalonSolov** [comment]
Only way would be to have wrappers for the math module's functions in A.

A lot of work to avoid a simple `import math`.

**Comment by spytheman** [comment]
Note, that even though both your Main and A modules import `math`, the code in `math` will be processed just once, i.e. you can import it whenever it is needed (and in fact you have to), V will ensure, that there will be no duplicate symbols because of it.
